cpp/src/Ice/Network.h

```diff
@@ -39,17 +39,12 @@ typedef int ssize_t;
 
 #if defined(__linux__) && !defined(ICE_NO_EPOLL)
 #    define ICE_USE_EPOLL 1
-#elif (defined(__APPLE__) || defined(__FreeBSD__) || defined(__FreeBSD_kernel__)) && TARGET_OS_IPHONE == 0 &&          \
-    !defined(ICE_NO_KQUEUE)
+#elif (defined(__APPLE__) || defined(__FreeBSD__) || defined(__FreeBSD_kernel__)) && TARGET_OS_IPHONE == 0)
```

Comment:
Removed ICE_NO_CFSTREAM and ICE_NO_KQUEUE because they are never used with our builds.